Components/HomeSidebar.tsx

```diff
@@ -20,16 +19,20 @@ const HomeSidebar = ({ pageNumber = 1 }: any) => {
   };
 
   return (
-    <div className="fixed top-0 flex  flex-col  p-4 w-fit h-screen justify-between ">
-      {/* <LogoWithName width={40} height={40} /> */}
-      <div className="flex flex-col space-y-4">
+    <div className="absolute top-0 flex  flex-col  p-4 justify-between w-[304px] h-screen left-[1px] bg-gradient-to-b from-[#e2e9f1] to-white">
+      {/* Top Part  */}
```

Comment:
No need of the comments in HTML part. Please remove all the comments from HTML in this file.